lib/takeaway.rb

```diff
@@ -0,0 +1,22 @@
+require_relative('./menu.rb')
+require_relative('./order.rb')
+
+class Takeaway
+  attr_reader :menu, :order
+
+    def initialize(menu:)
+      @order = Order.new
+      @menu = menu
+    end 
+
+    def print_menu
+      @menu.print
+    end
+
+    def add_to_order(dish, quantity)
+      @order.my_dishes << {dish => quantity}
+      puts "#{quantity} #{dish} added to your order!"
+    end 
+
+end 
+
```

Comment:
I like that you used many different classes/module to separate the responsibilities of the program. I think this will make it easier to expand the functionality of the program in the future.